apps/dashboard/src/components/primitives/form/avatar-picker.tsx

```diff
@@ -35,79 +35,82 @@ type AvatarPickerProps = {
   name: string;
   value: string;
   onChange?: (value: string) => void;
+  onPick?: (value: string) => void;
 };
 
-export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ name, value, onChange }, ref) => {
-  const { step } = useStep();
-  const variables = useMemo(() => (step ? parseStepVariablesToLiquidVariables(step.variables) : []), [step]);
-  const [isOpen, setIsOpen] = useState(false);
-  const { error } = useFormField();
+export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(
+  ({ name, value, onChange, onPick }, ref) => {
+    const { step } = useStep();
+    const variables = useMemo(() => (step ? parseStepVariablesToLiquidVariables(step.variables) : []), [step]);
+    const [isOpen, setIsOpen] = useState(false);
+    const { error } = useFormField();
 
-  const handlePredefinedAvatarClick = (url: string) => {
-    onChange?.(url);
-    setIsOpen(false);
-  };
+    const handlePredefinedAvatarClick = (url: string) => {
+      onPick?.(url);
+      setIsOpen(false);
+    };
```

Comment:
I had to differentiate the onChange from avatar image pick, because on pick the API should be called